public/index.html

```diff
@@ -93,7 +77,7 @@
         if (d.getElementById(id)) return;
         js = d.createElement(s);
         js.id = id;
-        js.src = '//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.10';
+        js.src = '//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.10'; // If this line throws error, disable adblocker.
```

Comment:
@yankeexe BTW, how does this work?

Reply:
@kabirbaidhya Based on this [document](https://developers.facebook.com/docs/javascript/quickstart/), it is used for social plugins from Facebook. This URL must be present in the host file of adblockers causing the error.